tests/php/sync/test_class.jetpack-sync-themes.php

```diff
@@ -262,22 +320,24 @@ public function test_update_themes_sync() {
 		$this->sender->do_sync();
 
 		$event_data = $this->server_event_storage->get_most_recent_event( 'jetpack_updated_themes' );
-		$themes = $event_data->args[0];
-
-		//Not testing versions since they are subject to change
-		$this->assertEquals( 'Twenty Seventeen', $themes['twentyseventeen']['name'] );
-		$this->assertEquals( 'https://wordpress.org/themes/twentyseventeen/', $themes['twentyseventeen']['uri'] );
-		$this->assertEquals( 'twentyseventeen', $themes['twentyseventeen']['stylesheet'] );
-		$this->assertEquals( 'Twenty Sixteen', $themes['twentysixteen']['name'] );
-		$this->assertEquals( 'https://wordpress.org/themes/twentysixteen/', $themes['twentysixteen']['uri'] );
-		$this->assertEquals( 'twentysixteen', $themes['twentysixteen']['stylesheet'] );
+		$themes     = $event_data->args[0];
+
+		// Not testing versions since they are subject to change.
```

Comment:
Is this still true now that we're supplying our own themes?

Same on line 351 below.

Reply:
more appropriate action would be a to-do or add check on version but that is not in scope.

Reply:
I'd be fine with changing the comment to a to-do.

Reply:
updated both instances of this comment to a TODO